canvg.js

```diff
@@ -231,8 +231,14 @@
 			}
 			else if (window.DOMParser)
 			{
-				var parser = new DOMParser();
-				return parser.parseFromString(xml, 'text/xml');
+				try {
+					var parser = new DOMParser();

```

Comment:
Reusing it caused issues in some browser(s). 
I can not remember on which browser I encountered the issue but I'll take a wild guess and say that Microsoft built it.